spec/features/tracks_spec.rb

```diff
@@ -60,12 +61,83 @@
       within('table#tracks') do
         expect(page.has_content?('Distribution')).to be true
         expect(page.has_content?('Events about our Linux')).to be true
-        expect(page.assert_selector('tr', count: 2)).to be true
+        expect(Track.count).to eq 1
+      end
+    end
+  end
+
+  shared_examples 'tracks' do
```

Comment:
These shared examples are all about managing / working with tracks, right?
Perhaps the name could be a little bit more specific?